docs/integrations/apps/guide/requirements.mdx

```diff
@@ -35,6 +35,7 @@ Don't reference competitor platforms in the app's listing information or dashboa
 * Apps that modify the checkout experience must use the BigCommerce [Checkout SDK](/docs/storefront/cart-checkout/checkout-sdk).
 * Apps that add another marketplace or sales channel to a store should make use of the [Channels Toolkit](/docs/integrations/channels/guide) and follow [Channel App Requirements](/docs/integrations/channels/guide/requirements).
 * Apps that create orders in the BigCommerce store need to properly mark accurate order source, payment method, and other order details.
+* Apps that are [compatible with multi-location inventory](https://support.bigcommerce.com/s/article/Multi-Location#app-support) must use the [Inventory API](/docs/store-operations/catalog/inventory-adjustments), not the [V3 Catalog API](/docs/rest-catalog), to adjust inventory in a store.
```

Comment:
Hey @bc-andreadao originally this ticket was intended to add new documentation about requirements for MLI compatibility, rather than updating the Marketplace approval requirements. Something similar to the MSF requirements doc we have here: https://developer.bigcommerce.com/docs/integrations/apps/multi-storefront

I'm hesitant to add this here because it's not a requirement for approval in the Marketplace and not applicable to all apps. Would it make more sense to add a tab under the "Buy online pick up in store" section here, since these are the APIs the MLI compatibility are related to? https://developer.bigcommerce.com/docs/integrations/buy-online-pick-up-in-store

Something like "App compatibility with multi locations" maybe? We can chat about this offline if it's easier, just give me a ping.